.gitignore

```diff
@@ -1,19 +1,44 @@
+.coverage
 *.egg-info
+*.log
 *.mo
-*.pyc
-*~
-.installed.cfg
-.mr.developer.cfg
+*.py?
+*.swp
+# dirs
 bin/
-coverage/
+buildout-cache/
 develop-eggs/
-downloads/
 eggs/
-parts/
-var/
+htmlcov/
 include/
 lib/
-Makefile
-.pydevproject
-.project
-.settings
+local/
+node_modules/
+parts/
+src/*
```

Comment:
I would just not ignore `src/*` and remove the `!src/collective` below

Reply:
I just used the `bobtemplates.plone` configuration:

https://github.com/plone/bobtemplates.plone/blob/05026f80839ecb84506a729082a63df093fff72c/bobtemplates/plone/addon/.gitignore.bob#L18

If you think this is important, you may want to open an issue on `bobtemplates.plone`. Can we leave it at that for now?

Reply:
Yeah same thing than above, I will open an issue now.
I would prefer to not had stuff that we already know is not sane, but anyway... no pressure

Reply:
https://github.com/plone/bobtemplates.plone/issues/437

Reply:
> plone/bobtemplates.plone#437

I'm going to do a test with this and see how it behaves. It's really bad to have to edit the .gitignore every time we add a file.

Reply:
@ale-rt now I am in doubt. With so many scripts generating files at the root of the project, we have to keep adding them to `.gitignore`. The return of  ignore `.*` may not be that bad. Then we could include only what we need. I for example had to modify gitignore now:

https://github.com/collective/collective.recaptcha/pull/19/commits/ef4972e7deba540d0ee4812981585e8c7e78fd35

@mauritsvanrees FYI

Reply:
The problem with wildcard is that you never know what happens to be ignored.
e.g. looking at ef4972e7deba540d0ee4812981585e8c7e78fd35 I do not get what you want to ignore, `.coverage` and what?
Anyway, that is just a personal prefence, do not take that as a blocker.

Reply:
@thet FYI

Reply:
I think you're right. Saying what you want to ignore is better than ignoring everything and saying what you want to versioned, frequent we often have to add ignored files.

Anyway, without the `. *`, It doesn't make sense the `!`